fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkimport/ChunkReader.java

```diff
@@ -102,8 +103,12 @@ public ChunkReader() {
 
     @Override
     public Object readItem() throws Exception {
+        // If the job is being stopped or in other status except for "started", then stop the read.
+        if (!stepCtx.getBatchStatus().equals(BatchStatus.STARTED)) {
+            return null;
+        }
         List<Resource> loadedFhirResources = new ArrayList<Resource>();
-        logger.info("readItem: get work item:" + importPartitionWorkitem + " resource type: " + importPartitionResourceType);
+        logger.fine("readItem: get work item:" + importPartitionWorkitem + " resource type: " + importPartitionResourceType);
```

Comment:
add an if logger, that way no creation of string to pass to fine logger

Reply:
done